src/main.rs

```diff
@@ -186,10 +192,19 @@ fn main() -> Result<()> {
         bail!("Cannot find mod directory. Please make sure the .mod file is correct.");
     }
     eprintln!("Using mod directory: {}", modpath.display());
+    eprintln!();
+
     set_mod_root(modpath.clone());
 
     let mut everything = Everything::new(&args.ck3.unwrap(), &modpath, modfile.replace_paths())?;
     everything.load_all();
+
+    // We must apply the --no-colour flag AFTER loading and applying the config,
```

Comment:
I'm afraid this won't work the way you expect, because everything.load_all() already emits parse errors and such before validation